src/py42/clients/_archiveaccess/restoremanager.py

```diff
@@ -117,6 +118,13 @@ def send_stream(
     ):
         num_files = sum([fs.num_files for fs in file_selections])
         num_bytes = sum([fs.num_bytes for fs in file_selections])
+        file_location = None
+
+        # Use expected request parameters for restoring to the original location.
+        if restore_path == PushRestoreLocation.ORIGINAL:
```

Comment:
I didn't see a point in having two parameters, one for the strategy and one for the restore path because if you want `ORIGINAL_LOCAITON`, the restore path parameter does nothing (and yet it is still required 🤔).  Therefore, I thought the best way would be to allow users to say "ORIGINAL_LOCATION" as a special value for `restore_path` and have py42 figure out the rest.